src/View/Artist/favorites.html.twig

```diff
@@ -24,7 +24,7 @@
 						</img>
 					</div>
 					<p class="artist-name d-flex align-items-center justify-content-center">{{ favorite.name_artist }}</p>
-					<form action="/artists/deleteFavorite" method="POST">
+					<form action="/artists/deleteFavorite2" method="POST">
```

Comment:
Be more explicit in your route you have artists and concerts. 2 and 3 are neither.

Reply:
update the name of redirected link for delete